src/app/models/form.server.model.ts

```diff
@@ -446,14 +446,14 @@ const compileFormModel = (db: Mongoose): IFormModel => {
           type: String,
           default: '',
         },
-        amount_cents: {
+        payment_amount: {
```

Comment:
Is there any specific reason to change the variable name here?

Reply:
I made the change to the variable name because I was using the same type for when the payment amount is in cents and when its in dollars and cents. I was a bit conflicted about this too when I was implementing it. 
As discussed w @timotheeg, I'll change it back! It's better for backward compatibility and understanding too.

Reply:
I've introduced `FormPaymentsDisplay` to separate typing for input values on the form and internal states (typed with `FormPayments`). So internal states will use `amount_cents` while floats can still be displayed on the form in `PaymentDrawer`.